fix(plymouth): avoid warning if /etc/plymouth/plymouthd.conf is not present

aafeijoo-suse opened the pull request:
openSUSE does not install `/etc/plymouth/plymouthd.conf`, so plymouthd falls back to `/usr/share/plymouth/plymouthd.defaults`.

```
$ dracut -f
sed: can't read /etc/plymouth/plymouthd.conf: No such file or directory
```

Follow-up for 7f41458b9d8350d08b0057e050d98b7a25c3e640

## Checklist
- [X] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

bdrung commented on modules.d/45plymouth/module-setup.sh:
```diff
@@ -12,7 +12,8 @@ pkglib_dir() {
     local config_file="$2"
-    sed -n "s/^${setting}= *\([^ ]\+\) */\1/p" "$config_file"
+    [[ -f $config_file ]] \
+        && sed -n "s/^${setting}= *\([^ ]\+\) */\1/p" "$config_file"
```
For better readability I suggest to use if-then-else:

```suggestion
    if [[ -f $config_file ]]; then
        sed -n "s/^${setting}= *\([^ ]\+\) */\1/p" "$config_file"
    fi
```

aafeijoo-suse committed:
fix(plymouth): avoid warning if /etc/plymouth/plymouthd.conf is not present

openSUSE does not install /etc/plymouth/plymouthd.conf, so plymouthd falls back
to /usr/share/plymouth/plymouthd.defaults

```
$ dracut -f
sed: can't read /etc/plymouth/plymouthd.conf: No such file or directory
```

Follow-up for 7f41458b9d8350d08b0057e050d98b7a25c3e640
